docs/04-guides/secure-video-playback.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Secure video playback
+description: Secure video playback with access control
+---
+
+import CodeBlock from "@theme/CodeBlock";
+import CodeBlockTabs from "@site/src/components/CodeBlockTabs";
+import CodeBlockTabItem from "@site/src/components/CodeBlockTabItem";
+
+# Secure video playback
+
+You can set a playback policy for a livestream to secure video playback.
+
+Your playback policy can be either:
+- `public` - No access control is applied to the asset
+- `jwt` - A JSON Web Token (JWT) is required to playback the asset
+
+If you set your stream's playback policy to JWT, you will 
+need to sign a token with a signing key and put it in the playback URL
+to be able to playback your stream.
+
+## Step 1: Create a livestream with a signed playback policy
+
+Create a livestream with a signed playback policy. The playback policy can 
+be specified when creating a new live stream, or can be added to an existing 
+[live stream](https://docs.livepeer.studio/reference/api/update-stream).
+
+## Step 2: Create a signing key
+
+A signing key is used to sign JWTs and are necessary to make a stream private.
+You can find more information about the response object [here](https://docs.livepeer.studio/reference/api/create-signing-keys).
+
+<CodeBlockTabs>
+<CodeBlockTabItem value="JavaScript" label="JavaScript">
+
+```js
+const response = await fetch(
+  "https://livepeer.studio/api/access-control/signing-key",
+  {
+    method: "POST",
+    headers: {
+      Authorization: `Bearer ${process.env.API_TOKEN}`,
+      "Content-Type": "application/json",
+    }
+  }
+);
+
+const responseObject = await response.json();
+```
+
+</CodeBlockTabItem>
+</CodeBlockTabs>
+
```

Comment:
Would add here a note that specify that the `privateKey` field is available only on creation, and cannot be retrieved later. WDYT?